stix_shifter_modules/splunk/stix_translation/encoders.py

```diff
@@ -5,7 +5,7 @@ def simple(value):
         return value
 
 def like(field, value):
-    return "like({}, \"{}\")".format(field, value)
+    return "like({}, \"%{}%\")".format(field, value)
```

Comment:
We shouldn't do this.  If the user wants wildcards at beginning and end, they would add them.

Reply:
In Splunk datasource, LIKE operator is not working as expected in case searching the substring without passing the wildcard (%).
e.g.
The STIX pattern search process:name LIKE 'test' is not giving the result even if there is data "testxyz" or "xxtest" in the datasource. Hence by adding "%" at start and end of the search input is giving correct substring search result. Even if the user performs pattern LIKE operator search by passing wildcard in search string, this change is not impacting its result. Let us know your comments on this.

Reply:
I think the Splunk behavior is the correct one.  STIX LIKE should act the same as SQL LIKE.  With your change, if the user wants to match e.g. "test" or "tost" with `LIKE 't_st` but not anything else, they couldn't.  It would also match "testxyz" and "xxtest".